lib/lookbook/preview_parser.rb

```diff
@@ -43,7 +43,6 @@ def callbacks
     def run_callbacks
```

Comment:
This will be called with the mutex held because yard runs the `after_parse_list` in the `YARD.parse` call:
- https://github.com/lsegal/yard/blob/5b93b3a4f15a52f3165edef112e718149d294c38/lib/yard/parser/source_parser.rb#L373
- https://github.com/lsegal/yard/blob/5b93b3a4f15a52f3165edef112e718149d294c38/lib/yard/parser/source_parser.rb#L114